package.json

```diff
@@ -61,31 +61,32 @@
     "@skyux/assets": "4.0.1",
     "@skyux/colorpicker": "4.3.0",
     "@skyux/config": "4.4.0",
-    "@skyux/core": "4.6.2",
+    "@skyux/core": "4.6.3",
     "@skyux/docs-tools": "4.9.1",
-    "@skyux/forms": "4.20.0",
+    "@skyux/forms": "4.20.1",
     "@skyux/http": "4.2.0",
     "@skyux/i18n": "4.2.1",
-    "@skyux/indicators": "4.10.0",
+    "@skyux/indicators": "4.11.0",
     "@skyux/inline-form": "4.1.1",
     "@skyux/layout": "4.7.1",
     "@skyux/list-builder": "4.0.4",
     "@skyux/list-builder-common": "4.0.1",
     "@skyux/list-builder-view-checklist": "4.0.1",
-    "@skyux/lists": "4.8.0",
+    "@skyux/lists": "4.9.0",
     "@skyux/lookup": "4.17.0",
     "@skyux/modals": "4.5.5",
     "@skyux/omnibar-interop": "4.0.1",
     "@skyux/popovers": "4.5.3",
     "@skyux/router": "4.2.2",
     "@skyux/select-field": "4.0.2",
-    "@skyux/tabs": "4.6.7",
-    "@skyux/theme": "4.17.0",
+    "@skyux/tabs": "4.6.8",
+    "@skyux/text-editor": "4.0.0-beta.0",
```

Comment:
Should this be bumped to `beta.1` since that has released? - totally minor.

Reply:
Yeah I can do that! Done.